lib/cambiatus_web/controllers/manifest_controller.ex

```diff
@@ -0,0 +1,61 @@
+defmodule CambiatusWeb.ManifestController do
+  use CambiatusWeb, :controller
+
+  alias Cambiatus.Commune
+
+  def get_subdomain(conn) do
+    with subdomain <- conn.host do
+      {:ok, subdomain}
+    else
+      nil ->
+        subdomain = conn.private.absinthe.context.domain
+        {:ok, subdomain}
+
+      _ ->
+        {:error, "No subdomain provided"}
+    end
+  end
+
+  def serve_manifest_json(conn, community) do
+    manifest =
+      community
+      |> manifest_template()
+      |> Poison.encode!()
+
+    json(conn, manifest)
+  end
+
+  def manifest_template(community) do
+    %{
+      name: "#{community.name} | Cambiatus",
+      short_name: community.name,
+      description: community.description,
+      start_url: community.website,
```

Comment:
This should be `community.subdomain`:
```suggestion
      start_url: community.subdomain,
```

https://developer.mozilla.org/en-US/docs/Web/Manifest/start_url